www/content/tutorial.md

```diff
@@ -820,18 +820,29 @@ Result.isOk (List.get ["a", "b", "c"] 1)
 ```
 
 ```roc
-Result.try (Str.toU64 "2") \idx -> List.get ["a", "b", "c", "d"] idx
-# returns `Ok "c"`
+Result.try (Str.toU64 "2") listGet
+
+listGet = U64 -> Result Str [OutOfBounds]
```

Comment:
If think there's a typo here: it should be `listGet :` instead of `listGet =`

Reply:
@Anton-4 this type is still here

Reply:
Oops, thanks for noticing, I forgot to test the code.